modules/ROOT/pages/single-resource-transaction.adoc

```diff
@@ -24,6 +24,10 @@ The example shown above defines a JMS message source that receives messages from
 
 Mule only commits messages that successfully pass through the complete flow. If at any point in the flow a message throws an error which is propagated (i.e. it is not handled by an `on-error-continue`), Mule rolls back the transaction. Refer to xref:on-error-scope-concept.adoc[On Error Scope] for details on how to handle errors.
 
+== Nested Transactions
+
+Nested transactions are not available for Single Resource transactions. Refer to xref:xa-transactions.adoc#nested-transactions[XA Transactions] for more details about nested transactions.
```

Comment:
You could avoid --> _about nested transactions_ at the end of the sentences. Since the title and paragraph starts with the same.